### PR TITLE
FIX: import mnemonic w/typo, 'try again' does nothing (closes #1286)

### LINKED ISSUE #1286: Bug: import menmonit, but with typo -> "failed to import" -> "try again" does nothhing
but it should redirect back to import screen with typed menmonics

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: screen/selftest.js [hunk 1]
```diff
@@ -35,6 +36,28 @@ export default class Selftest extends Component {
     let isOk = true;
 
     try {
+      /* try {
```
Comment (Overtorment) [inline screen/selftest.js:39]: look @marcosrdz I committed debug stuff, and you approved that PR. I feel ashamed and you should too

Comment (marcosrdz) [reply screen/selftest.js]: Thats acceptable cause Selftest got modified in the dark mode branch. That wasnt going to make it in

Comment (Overtorment) [reply screen/selftest.js]: not if I wont merge darkmode kek

Comment (marcosrdz) [reply screen/selftest.js]: The wrath of Nuno would be strong

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tn10jzbdphg1m2k8267d19bpgw

[download apk](http://filestorage.bluewallet.io:1488/1594052608.apk)

File: screen/selftest.js (hunk 1 above)
Comment (Overtorment) [reply screen/selftest.js]: breaking master is not acceptable in any case

**Pull request merged**